shapes/produce_shapes.py

```diff
@@ -223,6 +223,9 @@ def main(args):
         tt.cuts.get("os").invert()
     if args.HIG16043:
         tt.cuts.remove("pt_h")
+    if args.embedding:
+        tt.cuts.remove("trg_doubletau")
+        tt.cuts.add(Cut("(pt_1>50 && pt_2>40)*(generatorWeight<1.0)+(pt_1>50 && pt_2>40 && trg_doubletau==1)*(generatorWeight>=1)", "trg_doubletau_emb"))
```

Comment:
Was ist der Grund hierfür?

Reply:
die hadronischen tau-trigger in den embedded events sind nicht gut beschrieben, das ist das einzige groessere issue das wir noch haben im moment. wir koennen den trg_doubletau deshalb fuer embedded events nicht anwenden und muessen die effizienz des triggers mit einem weight skalieren